spec/song_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "verses"
+require "song"
+
+describe Song do
+
+  before(:each) do
+    @verses = spy(Verses)
+    @song = Song.new(@verses)
+  end
+
+  # it "sings verse once, given one bottle" do
+  #   verses = spy(Verses)
+  #   song = Song.new(verses)
+  #   expect(song.create_song(1)).to include("1 green bottle", "no more bottles")
+  # end
```

Comment:
I try not to commit commented out code. If not using this, I'd probably delete it before merging :)